code/modules/hydroponics/grown/ambrosia.dm

```diff
@@ -67,8 +67,8 @@
 /obj/item/seeds/ambrosia/gaia
 	name = "pack of ambrosia gaia seeds"
 	desc = "These seeds grow into ambrosia gaia, filled with infinite potential."
-	icon_state = "seed-ambrosia_gaia"
-	species = "ambrosia_gaia"
+	icon_state = "seed-ambrosiagaia"
```

Comment:
I don't think you want to change this, unless you actually change the name of the icon state itself in the image files!
```suggestion
	icon_state = "seed-ambrosia_gaia"
```